lib/membrane_hackney/source.ex

```diff
@@ -67,8 +70,11 @@ defmodule Membrane.Hackney.Source do
                 default: false
               ]
 
+  @spec get_resource_tag() :: atom()
+  def get_resource_tag(), do: @resource_tag
```

Comment:
as far as I can see, that function is used only in tests - perhaps it would be better to remove it and use `:hackney_source_resource` atom in an inline manner in tests?